cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,54 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
-
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+  it('should display the total number of pages when new rows are added', () => {
```

Comment:
add actions to this test:
1. Click the next button
2. Assert that page 2 is displayed
3. Click previous button
4. Assert that page 1 is displayed

Reply:
Add actions: 
Click previous button
Assert that page 1 is displayed